resources/PhoneNumberMetadata.xml

```diff
@@ -27282,13 +27938,6 @@
           )[2-9]\d{6}
         </nationalNumberPattern>
       </personalNumber>
-      <!-- 710 range is assigned to federal government of the US. As these are service numbers
-           (non geographical) supporting them here. -->
```

Comment:
👋  @penmetsaa Why was this removed?

(I didn't know it existed until today - it turns out that people from North Dakota (area code 701) have a hard time typing their phone numbers correctly)